src/main/java/com/google/api/codegen/config/ResourceDescriptorConfig.java

```diff
@@ -75,6 +76,19 @@
   /** The entity name for the resource config. */
   public abstract String getDerivedEntityName();
 
+  public static ResourceDescriptorConfig getWildcardResource(ProtoFile protoFile) {
+    return new AutoValue_ResourceDescriptorConfig(
+        /* isDefinedAtMessageLevel= */ false,
```

Comment:
super not important: please format these (some have spaces after `/*` some don't)

Reply:
Done.